examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/CastingServerBridge.h

```diff
@@ -186,6 +186,15 @@
  */
 - (void)disconnect:(dispatch_queue_t _Nonnull)clientQueue requestSentHandler:(nullable void (^)())requestSentHandler;
 
+/*!
+ @brief Purge data cached by the Matter casting library
+
+ @param clientQueue Queue to invoke callbacks on
+
+ @param responseHandler Called when purgeCache completes
+ */
+- (void)purgeCache:(dispatch_queue_t _Nonnull)clientQueue responseHandler:(nullable void (^)(MatterError * _Nonnull))responseHandler;
+
 /**
```

Comment:
any reason the doc format is different on this one line?

```suggestion
/*!
```

Reply:
Most likely a Copy-paste error. Fixed.